jsondb.js

```diff
@@ -102,20 +107,27 @@ export const methodPut = ({ ...props }) => {
   const path = restfulLensPath(lensPath, json);
   const parentPath = restfulLensPath(lensPath.slice(0, -1), json);
   const parentObj = U.path(parentPath, json);
-  const targetObj = U.path(path, json);
+  const targetObj = !U.isEmpty(path) ? U.path(path, json) : null;
```

Comment:
Shouldn't be necessary since U.path returns null if nothing is found.

Reply:
This is done to ensure that `targetObj` won't be the root object - it will happen in case lens path doesn't exist (in which case `path` value will be `[]`) and `U.path` will then return the whole JSON.